src/campsite-info/components/camp-rating/camp-rating.utils.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { tooltipLabels } from './camp-rating.constants';
+import { Box } from '@mui/material';
+import { tooltipStyles } from './camp-rating.styles';
+
+export const GetLabelText = (value: number) => {
+  return `${value} Star${value !== 1 ? 's' : ''}, ${tooltipLabels[value]}`;
+};
+
+export const DisplayLabelText = (value: number, hover: number) => {
```

Comment:
this thing isn't actually a util, it's a component

Reply:
Good catch! I separated it out into a component file which is nested in the camp-rating folder